tests/storage_test.go

```diff
@@ -422,6 +428,80 @@ var _ = Describe("Storage", func() {
 			})
 		})
 
+		Context("With smaller than requested PVCs", func() {
+
+			var pvc string
+
+			BeforeEach(func() {
+
+				// prepare storage with fix small size (20Mi) where fs already has > 10 % some overhead
+				out, err := exec.Command("/bin/sh", "-c", `
```

Comment:
How about creating such a file-system in a privileged pod, then use mount propagation to mount that file-system via a hostDisk into the vmi pod?

Reply:
Here the docu: https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation and here where we use it in a POC: #1565

Reply:
I already tried to do this in disk-images-provider, similar to creating the cirros block device there, but it didn't work... so at least for this PR I'd prefer to keep it his way